### DISCUSSION TITLE
How Slef host ?

### DISCUSSION THREAD

**Discussion by AnonymousWebHacker** [amanharwara/altus#137] [Q&A]
Is there a way to leave it as a web service? I mean self-host ?
 and how? I can't find wiki or anything

**Comment by AnonymousWebHacker** [comment]
Resolv
https://github.com/amanharwara/altus/discussions/118

**Comment by amanharwara** [comment]
> Is there a way to leave it as a web service? I mean self-host ?

Hi. I'm not sure how you could use an Electron app as a web service. I think you might be able to use something like Docker. Although I'm really not aware how you'd actually do that.
Being a web service really isn't in the scope of this app, and I'm not even sure how one would make this into a web service since it requires specific Electron APIs to work.

**Reply by AnonymousWebHacker** [reply]
I can't host it for me as a web service?

**Reply by amanharwara** [reply]
It's not a web service, it's a desktop application.

**Reply by AnonymousWebHacker** [reply]
I would like to see a web service like this

**Reply by amanharwara** [reply]
You could just use WhatsApp Web in your browser
